### DISCUSSION TITLE
[Projects Beta] Adding a View description section

### DISCUSSION THREAD

**Discussion by takuyakikuchi** [community/community#6429] [Projects and Issues]
# Suggestion

Adding a View description section.
![Image 2021-10-17 at 10 16 00 AM](https://user-images.githubusercontent.com/51087054/137606530-076c70fe-f4c9-4677-9a1a-12812b1e9462.jpg)

# Possible benefits

It can be used for giving quick introduction of the view to team members.

**Comment by rraadd88** [comment]
Not an answer but a related suggestion.

Similarly, the "tiles" on the board layout _for which description exists_ could be marked as shown below:

![image](https://user-images.githubusercontent.com/9945034/220687363-b0decd09-7826-45d8-b2df-79c80aa540ac.png)
